Competition2017/src/competition/operator_interface/OperatorCommandMap.java

```diff
@@ -24,6 +30,18 @@ public void setupMyCommands(
     */
     
     @Inject
+    public void setupLauncherCommands(
+            OperatorInterface oi,
+            ShooterSubsystem shooterSubsystem,
+            XPropertyManager propertyManager
+            ) {
+        oi.leftButtons.getifAvailable(6).whenPressed(new IncreaseShooterPowerCommand(shooterSubsystem.getLeftShooter()));
```

Comment:
As long as you've merged master recently and the tests don't warn you of double allocation, I'm happy.